src/Raven.Studio/wwwroot/App/views/manage/certificates.html

```diff
@@ -28,7 +28,7 @@ <h2 class="text-warning"><i class="icon-unsecure"></i> Authentication is off</h2
                 </div>
             </div>
             <div class="text-center margin-top margin-lg">
-                For more information please see <a href="https://ravendb.net/docs" target="_blank">RavenDB 4.0 Documentation</a>.
+                For more information please see <a href="https://ravendb.net/docs" target="_blank">RavenDB 4.1 Documentation</a>.
```

Comment:
This should be 
`https://ravendb.net/docs/article-page/4.0/csharp` in 4.0 
and
`https://ravendb.net/docs/article-page/4.1/csharp` in 4.1